test/rbs/test/type_check_test.rb

```diff
@@ -120,19 +119,68 @@ def test_sampling_handling
     SignatureManager.new do |manager|
       manager.build do |env|
         builder = DefinitionBuilder.new(env: env)
+        
+        begin assert '#sample'
```

Comment:
These `begin`s are useless.

Reply:
You can clean up the tests because you need only two cases, `nil` and an Integer.